test/sql/aggregate/aggregates/test_state_export.test

```diff
@@ -0,0 +1,137 @@
+# name: test/sql/aggregate/aggregates/test_state_export.test
+# description: Test the state export functionality
+# group: [aggregates]
+
+statement ok
+PRAGMA enable_verification
+
+# TODO make this work with ze optimizer
+statement ok
+PRAGMA disable_optimizer
+
+statement ok
+create table dummy as select range % 10 g, range d from range(100);
+
+# ungrouped aggr
+
+query IIIIII nosort res0
+SELECT count(*), count(d), sum(d), avg(d)::integer, min(d), max(d) FROM dummy;
+----
+100	100	4950	49	0	99
+
+query IIIIII nosort res0
+SELECT finalize(count(*) EXPORT STATE), finalize(count(d) EXPORT STATE), finalize(sum(d) EXPORT STATE), finalize(avg(d) EXPORT STATE)::integer, finalize(min(d) EXPORT STATE), finalize(max(d) EXPORT STATE)  FROM dummy;
```

Comment:
We should add some tests with:

* Storing the export/combine/finalize as part of a view on disk
* Storing the aggregate state on disk persistently
* More types of aggregate functions

Reply:
Aye

Reply:
Note to self, tests with aggregates that read no data or only NULLs